packages/mobile/src/utils/accountChecker.ts

```diff
@@ -13,9 +15,10 @@ export function* checkAccountExistenceSaga() {
   const gethAccounts: string[] = yield wallet.getAccounts()
   const reduxAddress: string = yield select(currentAccountSelector)
   if (!reduxAddress && gethAccounts.length > 0) {
-    // TODO: Try to recover access to the account.
+    const account = gethAccounts[0]
```

Comment:
Is it guaranteed that the right account we want will always be the first one?

Reply:
I'm not sure really, but I don't know why we would have another? In the tests I've made there's only one and I don't know of a way to have more than one, though I guess it's possible. What would you expect the behavior to be if there's more than one?

Reply:
yes not too sure either, let's go with this for now.